main.py

```diff
@@ -80,6 +88,19 @@ def run_rufas(
         )
 
 
+def reload_var_pool(vars_file_path: str) -> None:
+    """Instantiates Output Manager and triggers reloading of the variables pool from the provided file path.
+
+    Parameters
+    ----------
+    reload_pool_path : str
+        The file path to the variables file to reload into the Output Manager's variables pool.
+    """
+    output_manager = OutputManager()
+    vars_file_path = Path("output/" + vars_file_path)
+    output_manager.reload_pool(vars_file_path)
```

Comment:
Don't modify `vars_file_path` here. This code assumes/forces the file is always in `output` dir. What if the user wants to have the file in somewhere else?
Also, we need to trigger some actions after loading the pool.
```suggestion
    output_manager.set_metadata_prefix("reloaded_data") #or something similar
    output_manager.reload_pool(vars_file_path)
    output_manager.dump_all_nondata_pools(
            r"output", exclude_info_maps, format_option
        )
        output_manager.save_variables(
            Path(r"output"),
            Path(r"output/output_filters/"),
            exclude_info_maps,
            produce_graphics,
            graphics_dir,
        )
```

Reply:
I've got these added. Was making changes as you did this review.